postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  plugins: [
+    require('autoprefixer'),
+    require('precss')
```

Comment:
Q: do we want to use [precss](https://github.com/jonathantneal/precss)? Or would it make more sense to go with Sass via [node-sass](https://github.com/sass/node-sass)? Or just stick with plain old CSS?

Backstory: I added PostCSS so we could get `autoprefixer` so thought about trying other PostCSS plugins out.

Reply:
Using Sass via node-sass as it's a bit easier to lint and auto format.